environment.yml

```diff
@@ -0,0 +1,37 @@
+#default environment for installing python package
+#Note: Use this
```

Comment:
the comments can be omitted.

Reply:
removed the comments and deleted the line with histogram